.pipelines/build-windows.cmd

```diff
@@ -0,0 +1,4 @@
+setlocal enabledelayedexpansion
+powershell.exe -ExecutionPolicy Unrestricted -NoProfile -WindowStyle Hidden -File "%~dp0..\build\windows\Makefile.ps1"
```

Comment:
what does ~dp0 do ? will it be the same in every machine for all of us ?

Reply:
> what does ~dp0 do ? will it be the same in every machine for all of us ?

Here is the explanation of ~dp0 https://stackoverflow.com/questions/5034076/what-does-dp0-mean-and-how-does-it-work. Yes, this will be same for all machines.